filebeat/module/haproxy/log/_meta/fields.yml

```diff
@@ -40,5 +40,23 @@
       description: raw_request_line is the complete HTTP request line, including the method, request and HTTP version string.
       type: text
   
-
+- name: default
```

Comment:
Don't namespace fields depending on the matching pattern, some fields can be common between them, like probably `frontend_name`.

Reply:
TCP format PR is also open and waiting and I think I did a better naming there. Let's move on to finish and close this two and then I'll quickly open a new one to write proper naming?

Reply:
After opening a new PR with both PRs, what prevents us on using a proper naming? I'd do it before merging.